app/org/sagebionetworks/bridge/models/activities/ActivityEventImpl.java

```diff
@@ -0,0 +1,51 @@
+package org.sagebionetworks.bridge.models.activities;
```

Comment:
Why is the implementation not just DynamoActivityEvent? Do we need another implementation?